apis/v1alpha1/generator.yaml

```diff
@@ -713,7 +713,49 @@ resources:
           is_ignored: true  
       OnStart:
         compare:
-          is_ignored: true  
+          is_ignored: true
+  Domain:
+    exceptions:
+      errors:
+          404:
+            code: ResourceNotFound
+      terminal_codes:
+        - ResourceNotFound
+        - InvalidParameterCombination
+        - InvalidParameterValue
+        - MissingParameter
+    hooks: 
+      sdk_create_post_set_output:
+        template_path: domain/sdk_create_post_set_output.go.tpl
+      sdk_read_one_post_set_output:
+        code: rm.customDescribeDomainSetOutput(ko)
+      sdk_update_pre_build_request:
+        template_path: common/sdk_update_pre_build_request.go.tpl
+      dk_delete_pre_build_request:
+        template_path: common/sdk_delete_pre_build_request.go.tpl
+      sdk_delete_post_request:
+        template_path: common/sdk_delete_post_request.go.tpl
+    fields:
+      DomainId:
+        is_read_only: true
+        print:
+          name: DOMAIN-ID
```

Comment:
We have generally not printed columns other than status, any reason for adding this here ?